src/AaveV3LlamaProposal_20230803/AaveV3LlamaProposal_20230803.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveMisc} from 'aave-address-book/AaveMisc.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+
+/**
+ * @title Reduce Llama Stream
+ * @author @marczeller - Aave-Chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xceae27bbaf42658a1b46baec664c66c09f9cba4f9452ed2d2bed6f6ce5c66e35
+ * - Discussion: https://governance.aave.com/t/arfc-cancel-llama-service-provider-stream/14137
+ */
+contract AaveV3LlamaProposal_20230803 is IProposalGenericExecutor {
+  address public constant LLAMA_RECIPIENT = 0xb428C6812E53F843185986472bb7c1E25632e0f7;
+  uint256 public constant ER_AAVE_STREAM = 100001;
+  uint256 public constant COLLECTOR_aUSDC_STREAM = 100003;
+
+  uint256 public constant AAVE_STREAM_AMOUNT = 283230000000000000000;
```

Comment:
No need to change anything now, but for visibility in the future I recommend doing things like 283.23e18 for example-- just to avoid any potential typos.